src/web/modules/stripe/basic/confirm-create-test-account.njk

```diff
@@ -0,0 +1,42 @@
+{% extends "layout/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block main %}
+  {% if systemLinkService %}
+    <div class="govuk-body">
+      <a class="govuk-link govuk-link--no-visited-state" href="/services/{{ systemLinkService }}">View service</a>
```

Comment:
As this is an action from the service detail page I'd recommend using the same pattern as other "action pages" which link back to the service they're acting on

<img width="593" alt="Screenshot 2021-02-15 at 16 59 48" src="https://user-images.githubusercontent.com/2844572/107975072-72263a80-6faf-11eb-8275-814447ed21d8.png">

Reply:
Updated!